packages/compass-connect/src/stores/index.js

```diff
@@ -307,6 +315,13 @@ const Store = Reflux.createStore({
    * validate instead the existing connection object.
    */
   async onConnectClicked() {
+    const { connectionModel } = this.state;
+    const trackEvent = {
+      is_favorite: connectionModel.isFavorite,
+      is_recent: this.state.connectionEntry === CONNECTION_ENTRY_RECENT,
```

Comment:
@mabaasit @mmarcon should we also have an `is_new` too?

Reply:
added it.